pkg/config/config.go

```diff
@@ -129,7 +129,21 @@ func (bp *Blueprint) Module(id ModuleID) (*Module, error) {
 		return nil
 	})
 	if mod == nil {
-		return nil, fmt.Errorf("%s: %s", errorMessages["invalidMod"], id)
+		err := InvalidModuleError{id}
+		clMod := ""
```

Comment:
Let's move it to separate function:
`suggestModuleIdFix(s string) (string, bool)`

Reply:
Done